README.md

```diff
@@ -118,7 +118,10 @@ The worker constructor takes the following options:
 
 * `core`, `queue`
 * `startupDelay` -- time, in ms, between creation of the worker and the first call to `queue.claimWork`
+* `interTaskDelay` -- time, in ms, after a task during which the worker is unavailable for a new task (idle timeout starts after the delay is complete)
 * `idleTimeout` -- maximum time the worker will remain idle
+* `capacity` -- number of tasks this worker can execute in parallel (must be 1 right now)
+* `utility` -- speedup factor of this worker (the worker's time to complete a task is duration ÷ utility) (must be 1 right now)
```

Comment:
Why is a speedup factor called `utility`? Why not `speedupFactor`?

Reply:
If only we could feed a parameter to workers to make them go faster!  Instead of an imperative configuration, it's a *characteristic* of a worker -- for example a if a "large" instance has utility 1, then a "4xlarge" might have utility 4 (or maybe we measure it at 3.5 or something).  It's a term jhford invented in aws-provisioner and I suspect we'll be bringing it back.

Reply:
Got it. Thanks.